templates/common-partials/script/searchbar.hbs

```diff
@@ -26,6 +26,9 @@ ANSWERS.addComponent("SearchBar", Object.assign({}, {
   {{#if verticalKey}}
     verticalKey: "{{{verticalKey}}}",
   {{/if}}
+  loadingIndicator: {
+    display: true
```

Comment:
The HHs may want the ability to toggle this on and off. So, instead of hard-coding it here, I think we should add the `true` default in the `ComponentSettings.SearchBar` of each template's `page-config`. That way, the HHs can see how to enable/disable the indicator if they want.

Reply:
oh yeah. I thought I would have it here and HHs would manually add this customization on whatever template they want. But explicitly have it on the template makes more sense. Updated!